### ISSUE TITLE
The finetuning weights on imagenet

### ISSUE THREAD

**Issue by YuetongFang** [keyu-tian/SparK#28]
Thanks for your wonderful work! The tutorial is fairly friendly for reproduction, but I also wonder if it is possible for us to get the fine-tuned weights on imagenet?

**Comment by keyu-tian** [comment]
yes, i think those would be ok in a couple of weeks. i'll work on them (weights & logs like https://github.com/keyu-tian/SparK/tree/main/downstream_d2#fine-tuned-resnet-50-weights-log-files-and-performance) once i finish my current project at hand.
